src/cascadia/WindowsTerminal/WindowsTerminal.manifest

```diff
@@ -14,6 +14,11 @@
         <!-- This will cause unpackaged activation failures in XAML Islands. -->
         <!-- (We use unpackaged activation in test scenarios.) See GH#10265. -->
         <maxversiontested Id="10.0.18362.0"/>
+
+        <!-- You apparently are expected to include all the maxversiontested
+        entries separately. They are treated like an array. This one turns on Segoe
+        UI Variable, GH #12452 -->

```

Comment:
Are you all sure this is true? I've looked into the source code behind this field and it doesn't indicate that it supports more than one value here.

I believe this value is stored at
```
HKEY_CLASSES_ROOT\Local Settings\Software\Microsoft\Windows\CurrentVersion\AppModel\Repository\Packages\*\OSMaxVersionTested
```
which only supports a single value. See `AppModelVersionP.hpp`.

Reply:
That may be true of the _appxmanifest_, but it is not true of the fusion manifest :smile:

Reply:
See MSFT-33672605 for more info!

Reply:
Nope. For future reference MSFT:33672605.